openra/fakes/engine_file_repository.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+from fs.tempfs import TempFS
+
+from openra.classes.file_location import FileLocation
+from openra.classes.release import Release
+from openra.services.engine_file_repository import ExceptionEngineFolderNotFound
+
+
+class FakeEngineFileRepository:
+
+    engine_exists:bool
+    imported:List[Release]
+
+    def __init__(self):
+        self.imported = []
+        self.engine_exists = False
+
+    def exists(self, release:Release):
+        return self.engine_exists
+
+    def get_path(self, release:Release):
+        if not self.engine_exists:
+            raise ExceptionEngineFolderNotFound(TempFS(), release, 'fake')
+
+        temp_fs = TempFS()
+        file = 'sample'
+        temp_fs.touch(file)
+        return FileLocation(
+            TempFS(),
+            '',
+            file
+        )
+
+
+    def import_appimage(self, release:Release, appimage_location:FileLocation):
+        self.imported.append(release)
+
+        temp_fs = TempFS()
+        file = 'sample'
+        temp_fs.touch(file)
+        return FileLocation(
+            TempFS(),
```

Comment:
One fake returns the tempFS stored in the local variable and two fakes create new ones inside the return statement. At least one of them should be wrong?

Reply:
It should be returning the `temp_fs` I think - will need to double check, but that seems to be the only way that makes sense to me atm

Reply:
Ok I've updated this now. I don't think that the parts that were being tested with this actually relied on it existing so it wasn't flagging up any issues.